sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/ServiceBusEndToEndTests.cs

```diff
@@ -296,9 +296,6 @@ private async Task TestMultiple<T>(bool isXml = false)
 
                 bool result = _topicSubscriptionCalled1.WaitOne(SBTimeoutMills);
                 Assert.True(result);
-
-                // ensure message are completed
-                await Task.Delay(2000);
```

Comment:
Interesting;  what bearing does this have on the test result?  Was this just an unneeded delay that we're cleaning up or was it involved in some flaky behavior?

Reply:
I think this was just left over from the T1 extension tests in which the same queue was used for each test. It wouldn't have caused flakiness but is now not needed.